move_group_name: panda_arm
planning_time: 25.0
planning_trials: 10
planners:
- mg1
- timebased
- probabilistic_hamp_time_based
- hamp_timebased

occ_center:
- 0
- 0
- 0.5

occ_radius:
- 1
- 1
- 0.5

filename_results: /home/hypatia/.ros/hamp_results.csv
